CIP-????/README.md

```diff
@@ -0,0 +1,67 @@
+- [P2WPKH](https://learnmeabitcoin.com/technical/script/p2wpkh/)
+- [P2SH](https://learnmeabitcoin.com/technical/script/p2sh/)
+- [P2WSH](https://learnmeabitcoin.com/technical/script/p2wsh/)
+
+Adding `RIPEMD-160` to Plutus would enhance the potential for cross-chain solutions between Cardano and Bitcoin blockchains and complements the set of primitives which we already have in that regard. It would allow for the verification of Bitcoin addresses and transactions on-chain. This addition enables also the verification of signed messages that identify the signer by the public key hash, which has not yet been witnessed on the Bitcoin blockchain.
+
+The RIPEMD-160 is not only relevant to Bitcoin - other chains like (Cosmos)[https://docs.cosmos.network/main/build/architecture/adr-028-public-key-addresses#legacy-public-key-addresses-dont-change] or (BNB)[https://docs.bnbchain.org/docs/beaconchain/learn/accounts/#address] also use it for address generation.
+
+## Specification
+This proposal aims to introduce a new built-in hash function `RIPEMD-160`.
+
+This function will be developed following the [`RIP-160`](https://homes.esat.kuleuven.be/~bosselae/ripemd160/pdf/AB-9601/AB-9601.pdf) specification and will utilize the [`cryptonite`](https://github.com/haskell-crypto/cryptonite/blob/master/Crypto/Hash/RIPEMD160.hs)
+
+Since `cardano-base` already relies on `cryptonite` in the context of [`keccak-256`](https://github.com/input-output-hk/cardano-base/blob/master/cardano-crypto-class/src/Cardano/Crypto/Hash/Keccak256.hs) we would like to expose `RIPEMD-160` through the same library, to facilitate its integration into Plutus.
+
+More specifically, Plutus will gain the following primitive operation:
+
+* `ripemd_160` :: ByteString -> ByteString
+
+The input to this function can be a `ByteString` of arbitrary size, and the output will be a `ByteString` of 20 bytes. 
+Note that this function aligns with the format of existing hash functions in Plutus, such as [blake2b_256](https://github.com/input-output-hk/plutus/blob/75267027f157f1312964e7126280920d1245c52d/plutus-core/plutus-core/src/Data/ByteString/Hash.hs#L25)
+
+## Rationale: how does this CIP achieve its goals?
+While the `RIPEMD-160` function might be implemented in on-chain scripts, doing so would be computationally unfeasible.
+
+The library, cryptonite, is not implemented by and under control of the Plutus team. However,
+* It is a library already used in the Plutus stack to expose KECCAK-256, and can be considered as a trustworthy implementation.
+* Its behaviour is predictable and computationally efficient. The cost of the function is linear with respect to the size of the message provided as input. This is the same behaviour that other hash functions exposed in plutus (blake, sha3, keccak-256) have.
+
+## Path to Active
+This CIP may transition to active status once the Plutus version containing the `ripemd_160` function is introduced in a node release and becomes available on Mainnet.
```

Comment:
Since this point refers to granting `Active` status, it should be moved into the list of _Acceptance Criteria_.